src/libraries/System.Formats.Tar/src/System/Formats/Tar/UstarTarEntry.cs

```diff
@@ -30,6 +30,19 @@ internal UstarTarEntry(TarHeader header, TarReader readerOfOrigin)
         public UstarTarEntry(TarEntryType entryType, string entryName)
             : base(entryType, entryName, TarEntryFormat.Ustar)
         {
+            _header._prefix = string.Empty;
+        }
+
+        /// <summary>
+        /// Initializes a new <see cref="UstarTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the Ustar format.
+        /// </summary>
+        public UstarTarEntry(TarEntry other)
+            : base(other, TarEntryFormat.Ustar)
+        {
+            if (other._header._format is TarEntryFormat.Ustar or TarEntryFormat.Pax)
+            {
+                _header._prefix = other._header._prefix;
+            }
```

Comment:
Should this constructor set `_header._prefix` to `string.Empty` if other isn't a Ustar or Pax, like the one above it does? It would seem to be an inconsistent state where one ctor sets it to empty, but the other leaves it `null`.

Reply:
I see what you mean. Sure, I'll set it to Empty if the other format isn't ustar or pax.